charts/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.7.3
+version: 5.7.4
```

Comment:
should now 5.7.5.

Reply:
Hi @zanhsieh - rebased and changed the version